test/cmake/target_js/out.txt

```diff
@@ -1,4 +1,5 @@
 prejs executed
+prejs executed
 lib_function
 lib_function2
-postjs executed
\ No newline at end of file
+postjs executed
```

Comment:
why is this a diff? some kind of evil whitespace/non-printing difference?

Reply:
Yeah the old file was missing a newline at EOF